File: threedollar-domain/src/main/java/com/depromeet/threedollar/domain/domain/store/StoreImageCreator.java
package com.depromeet.threedollar.domain.domain.store;

import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class StoreImageCreator {

    public static StoreImage create(Long storeId, Long userId, String url) {
        return StoreImage.builder()
            .storeId(storeId)
            .userId(userId)
            .url(url)
            .build();
    }

}

File: threedollar-domain-rds/src/testFixtures/java/com/depromeet/threedollar/domain/rds/user/domain/store/StoreImageCreator.kt
package com.depromeet.threedollar.domain.rds.user.domain.store

import com.depromeet.threedollar.domain.rds.user.domain.TestFixture

@TestFixture
object StoreImageCreator {

    @JvmStatic
    fun create(
            store: Store,
            userId: Long,
            url: String
    ): StoreImage {
        return StoreImage.builder()
            .store(store)
            .userId(userId)
            .url(url)
            .build()
    }

}
